File: Lab 1/lab1.py
# import libraries
import time
import RPi.GPIO as GPIO

# define variables
ledPin =  3

def main():
	GPIO.setmode(GPIO.BOARD)
	GPIO.setup(ledPin, GPIO.OUT)
	GPIO.output(ledPin, GPIO.HIGH)

	for i in range(10):
		GPIO.output(ledPin, GPIO.HIGH)	# turn LED on
		time.sleep(5)			# wait 5 seconds
		GPIO.output(ledPin, GPIO.LOW)	# turn LED off
		time.sleep(5)			# wait 5 seconds

if __name__ == "__main__": main()

File: Lab 1/lab1.c
// import libraries
#include <stdio.h>
#include <stdlib.h>
#include <wiringPi.h>

// variable declaration
const int ledPin = 8;

int main(){
	// intialize wiringPi using Broadcom pin numbers
	wiringPiSetup();

	// set pins to input/output
	pinMode(ledPin, OUTPUT);

	int i = 0;
	for (i = 0; i < 10; i++){
		digitalWrite(ledPin, HIGH);	// turn LED on
		delay(5000);			// wait 5 seconds
		digitalWrite(ledPin, LOW);	// turn LED off
		delay(5000);			// wait 5 seconds
	}
}
